src/sentry/taskworker/worker.py

```diff
@@ -254,14 +256,15 @@ def __init__(
         self._concurrency = concurrency
         self.client = TaskworkerClient(rpc_host, num_brokers)
         self._child_tasks: multiprocessing.Queue[TaskActivation] = mp_context.Queue(
-            maxsize=queue_size
+            maxsize=concurrency
```

Comment:
From my sandbox testing, changing this from 1 -> 8 increased the task execution time (time a task spends in taskworker) from a max of 3s to 5s (though, this was also coupled with slow results queue flushing). It might be beneficial to make this change in a separate PR so we can observe the effects of the results queue change and change child queue size separately.

Reply:
How about we have a CLI option for this queue in the short term, then we can experiment with different values more quickly via k8s changes.